### ISSUE TITLE
Setup the phimpe app with a vanilla install of Wordpress and outline steps

### ISSUE THREAD

**Issue by mariobehling** [fossasia/phimpme-wordpress#1]
The Wordpress version and modules in this repository are outdated. The goal is to use the phimpme app as much as possible with a standard vanilla Wordpress. Therefore, please:
- [ ] Install a standard Wordpress from http://wordpress.org and test or implement the app with the standard install
- [ ] Document how to get the app working with Wordpress in the Readme.md
- [ ] Delete any files/folders no longer required in this repository

**Comment by pa1pal** [comment]
@mariobehling The plugins which are used in wordpress are not supported in newer version of wordpress i.e 4.6.1.
Tested on previous version 3.5.1

**Comment by mariobehling** [comment]
@pa1pal Yes, this is an issue, cause people will not install an old wp version to be compatible with the app.

**Comment by pa1pal** [comment]
There are no new similar plugins there.

**Comment by mariobehling** [comment]
I think, it should be possible to upload picture through the standard plugin. Please check out the Wordpress mobile app. This was not possible in the past. So, I guess there is no need for dedicated plugins anymore. Cheers.
